README.rst

```diff
@@ -71,7 +71,32 @@ To check if everything is ready to go, you can always test both fixtures:
 Configuration
 =============
 
-You can define your own path for elasticsearch logs directory. There are three ways to achieve this:
+You can define your own paths and/or variables for elasticsearch:
```

Comment:
> You can define your settings in three ways, it's fixture factory argument, command line option and pytest.ini configuration option.

Then it would be good to describe in what order these settings are being resolved.

Table headers should be set accordingly, With first column header's noting that it's Elasticsearch option.